package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "enrise-logger",
   "description": "Logger used within Enrise projects and module's",
-  "version": "0.2.1",
+  "version": "1.0.0",
```

Comment:
As long as we're not 1.0, I'd really prefer to increase just the minor version #. Anything before 1.0 still allows breaking changes until we decide it stable and feature complete enough.